gear/gear/auth.py

```diff
@@ -53,8 +71,11 @@ async def userdata_from_rest_request(request):
     auth_header = request.headers['Authorization']
     session_id = maybe_parse_bearer_header(auth_header)
     if not session_id:
-        return session_id
-    return await _userdata_from_session_id(auth_header[7:], request.app['client_session'])
+        return None
+
```

Comment:
Where did the `auth_header[7:]` go?

Reply:
I'm pretty sure `session_id == auth_header[7:]`, since the slice should be taking out `Bearer: `